src/System.Management.Automation/engine/runtime/CompiledScriptBlock.cs

```diff
@@ -1774,7 +1774,7 @@ internal static string CheckSuspiciousContent(Ast scriptBlockAst)
             return null;
         }
 
-        class SuspiciousContentChecker
+        static class SuspiciousContentChecker
```

Comment:
@PaulHigin @TravisEz13 Can we make the class static?

Reply:
The class contains only static members.

Reply:
@iSazonov Should I revert?